superset-frontend/src/components/Icon/index.tsx

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 import React, { SVGProps } from 'react';
+import { ReactComponent as BinocularsIcon } from 'images/icons/binoculars.svg';
```

Comment:
These icon additions may conflict with https://github.com/apache/incubator-superset/pull/11033 so we should probably coordinte with @rusackas

Reply:
Merged that PR! Please feel free to rebase and use the new binoculars icon (_and many more_) from master.